packages/sha/package.js

```diff
@@ -1,9 +1,12 @@
 Package.describe({
-  summary: "SHA256 implementation",
-  version: "1.0.4-plugins.0"
+  name: 'sha256',
+  version: '1.0.4-plugins.0'
+  summary: 'SHA256 implementation',
+  git: 'https://github.com/meteor/meteor/tree/devel/packages/sha',
+  documentation: 'README.md'
 });
 
 Package.onUse(function (api) {
   api.export('SHA256');
-  api.addFiles(['sha256.js'], ['client', 'server']);
+  api.addFiles('sha256.js', ['client', 'server']);
```

Comment:
`package.js` had `['client', 'server']` to begin with; I had just left it alone. Gone now.